lib/commands/-command.js

```diff
@@ -6,7 +6,7 @@ module.exports = Command.extend({
   init() {
     process.env.EMBER_CLI_ELECTRON = true;
 
-    if (this._super && this._super.init) {
+    if (this._super && this._super.init && this._super.init !== this.init) {
```

Comment:
Ah, okay. I'm running ember-cli `2.4.3`. My fault for not even considering that...